packages/x-components/src/x-modules/facets/service/types.ts

```diff
@@ -30,13 +31,18 @@ export interface FacetsService {
    *
    * @param facetIds - An optional list of facets ids from whom deselect the filters.
    */
-  clearFilters(facetIds?: Array<Facet['id']>): void;
+  clearFilters(facetIds?: Array<Facet['id']>, metadata?: Dictionary): void;
```

Comment:
Should we change the type of metadata? right now if you wanna send things in the metadata, you have to know the implementation to know that you have to send `keepSticky` to make it sticky because the types are not helping.

Does make it sense to create an interface to define the metadata type?

```
export interface FiltersMetadata {
  keepSticky: boolean;
}
```